cirrus/google_cloud/manager.py

```diff
@@ -306,6 +306,60 @@ def get_project_organization(self):
             org = info["parent"]["id"]
         return org
 
+    def get_project_members(self):
+        """
+        Get all the members given a project
+
+        Returns:
+            list(str): list of members
+
+            .. code-block:: python
+
+            {
+              "version": 1,
+              "etag": "BwVvrr5i9Jc=",
+              "bindings": [
+                {
+                  "role": "roles/compute.serviceAgent",
+                  "members": [
+                    "serviceAccount:service-xxxx@compute-system.iam.gserviceaccount.com"
+                  ]
+                },
+                {
+                  "role": "roles/container.serviceAgent",
+                  "members": [
+                    "serviceAccount:service-xxxx@container-engine-robot.iam.gserviceaccount.com"
+                  ]
+                },
+                {
+                  "role": "roles/owner",
+                  "members": [
+                    "user:test@planx-pla.net"
+                  ]
+                }
+              ]
+            }
+        """
+        members = []
+
+        api_url = _get_google_api_url(
+                "projects/" + self.project_id + ":getIamPolicy", GOOGLE_CLOUD_RESOURCE_URL)
+        response = self._authed_request("POST", api_url)
+        if response.status_code != 200:
+            return members
+
+        bindings = response.json().get('bindings',[])
+        for binding in bindings:
+            if not isinstance(binding,dict):
+                continue
+            users = binding.get('members',[])
```

Comment:
Minor style complaint, there should be a single space between arguments. should be `.get('members', [])`